toxcore/tox.c

```diff
@@ -13,6 +13,7 @@
 #include "tox.h"
 
 #include <assert.h>
+#include <stdarg.h>
```

Comment:
Is this include actually needed? I don't see any new lines needing stuff from stdarg.h in tox.c.

Reply:
removed